src/status_im/utils/pre_receiver.cljs

```diff
@@ -8,10 +8,6 @@
 (defn- add-message-mock [{:keys [id clock-value] :as msg}]
   (log/debug "add-message-mock:" id clock-value))
 
-(defn- delay-message [msg out ms]
-  (async/go (async/<! (async/timeout ms))
```

Comment:
Wow, I have no idea what this go statement is doing here. Probably a leftover from REPL experimentation. It should indeed be gone!